lib/Extension/CodeTransformExtra/Rpc/OverrideMethodHandler.php

```diff
@@ -121,6 +121,7 @@ private function parentClass(ReflectionClass $class)
 
     private function methodChoices(ReflectionClass $parentClass)
     {
+        /** @todo filter methods already implemented in the current class */
```

Comment:
:+1:

Reply:
could _possibly_ add a new API method in worse reflection, e.g. `$methods = $parentMethods->excluding($classMethods->belongingTo($className))`

Otherwise eqaually fine to quickly implement here (not necessarily in this PR)

Reply:
Is this the coding style comment you were referencing earlier ?
The `/** @todo` instead of `// TODO ` ?
If yes I will change it, only saw the `// TODO` afterward

Reply:
No, but "//” would be more consistent, I was referring to the setMultipleAllowed method call